server/server.go

```diff
@@ -128,6 +113,61 @@ func NewServer(cfg *Config) (Server, error) {
 	return s, nil
 }
 
+func createCommittee(cfg *Config) (committee.Committee, error) {
```

Comment:
let's move createCommitee to committee package, and rename it to 
`NewArchiveFromConfig(cfg db.Config, startHeight uint64, heightInterval) (Archive, error)`